sourcemash/items/models.py

```diff
@@ -11,7 +11,7 @@ class Item(db.Model):
     feed_id = db.Column(db.Integer, db.ForeignKey('feed.id'))
     category_1 = db.Column(db.String(100))
     category_2 = db.Column(db.String(100))
-
+    totalVotes = db.Column(db.Integer)
```

Comment:
If you can upvote and downvote, totalVotes really represents the voteSum, not the voteCount. We should rename this accordingly. Also, do we want to maybe add a voteCount? It would be helpful to use votes as a way to pick the best article to insert, but we'll need a tally of votes to do this.